ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "CaseEventID": "CREATE_CLAIM",
+    "UserRole": "caseworker-cmc-solicitor",
+    "CRUD": "CRUD"
```

Comment:
Wouldn't `CR` be enough?

Reply:
yep you're probably right, a lot of this I just copied over from the existing json files! I will go through and clean these up